src/reducers/pause.js

```diff
@@ -165,7 +166,7 @@ function update(state: PauseState = State(), action: Action): PauseState {
       };
 
     case "NAVIGATE":
-      return { ...state, debuggeeUrl: action.url };
+      return { ...state, ...emptyPauseState, debuggeeUrl: action.url };
```

Comment:
This seems like an unrelated change. Is it somehow necessitated by the changes to the `COMMAND` action?

Reply:
It is something that happened before, but this change makes it more explicit. It is helpful to do it here to prevent future regressions. Also simpler to test